staking/contracts/contracts.py

```diff
@@ -301,12 +306,12 @@ def config(
 ) -> Expr:
     return Seq(
         is_admin(),
-        App.globalPut(Bytes("P"), Not(Not(paused.get()))),
```

Comment:
Do we need a double negation here for a specific purpose, e.g. casting?

Reply:
Currently, the signature of config here differs from that of [the Teal's ABI Contract](https://github.com/algochoi/django-staking-demo/blob/algochoi/contract-review/staking/contracts/staking.json#L25-L33).

I suspect this is the correct version.